pkg/k8s/endpoints.go

```diff
@@ -196,6 +197,7 @@ func ParseEndpointSliceV1Beta1(ep *slim_discovery_v1beta1.EndpointSlice) (Endpoi
 				if option.Config.EnableK8sTerminatingEndpoint {
 					if sub.Conditions.Terminating != nil && *sub.Conditions.Terminating {
 						backend.Terminating = true
+						metrics.TerminatingEndpointsEvents.Inc()
```

Comment:
for EndpointSliceV1 as well ? 

https://github.com/cilium/cilium/blob/7e07d55efdd8f721f2caf15fd4212336a6478129/pkg/k8s/endpoints.go#L283